src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessarysemicolonaftertypememberdeclaration/InputUnnecessarySemicolonAfterTypeMemberDeclarationNullAst.java

```diff
@@ -0,0 +1,23 @@
+/*
+UnnecessarySemicolonAfterTypeMemberDeclaration
+tokens = (default)CLASS_DEF, INTERFACE_DEF, ENUM_DEF, ANNOTATION_DEF, VARIABLE_DEF, \
+         ANNOTATION_FIELD_DEF, STATIC_INIT, INSTANCE_INIT, CTOR_DEF, METHOD_DEF, \
+         ENUM_CONSTANT_DEF, COMPACT_CTOR_DEF, RECORD_DEF
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.unnecessarysemicolonaftertypememberdeclaration;
+
+public class InputUnnecessarySemicolonAfterTypeMemberDeclarationNullAst {
+    public static void main(String[] args) { // ok
```

Comment:
Please remove all ok comments.
I know that some check demands file to have at least one, please add it over class